### DISCUSSION TITLE
Repo collections

### DISCUSSION THREAD

**Discussion by picocodes** [community/community#35643] [Other Feature Feedback, Questions, & Ideas] [answered]
I've starred close to 1000 repos and most of them get lost behind all the stars. It would be great if you could add a feature to group stars by collection / list / category. Similar to how we can group Chrome bookmarks into folders.

**Comment by fullstop125** [comment]
hi @picocodes  This is a really cool idea ..you could probably add the enhancement label to this so that the dev team could see this request.

**Comment by igor-h** [comment, marked as answer]
Hi @picocodes 

Actually you can do that with the recently feature **GitHub** Implemented.

Only what you need to do is:

- Go to your profile
- open Stars Tab
- Create new list

![image](https://user-images.githubusercontent.com/39650812/194900123-f5841fc4-22b7-4c06-98a9-d18c871fd0c5.png)

![image](https://user-images.githubusercontent.com/39650812/194900212-d28bd5fe-b9ab-4490-a050-dfb8b4148fe5.png)


If this was helpful mark my comment as _Resolved_.

Thanks, Igor. 👍🏽

**Reply by picocodes** [reply]
Thanks, didn't know about that!
